tests/unit/src/kakarot/instructions/test_system_operations.cairo

```diff
@@ -9,6 +9,9 @@ from starkware.cairo.common.uint256 import Uint256
 from starkware.starknet.common.syscalls import deploy, get_contract_address
 from starkware.cairo.common.math import split_felt, assert_not_zero, assert_le
 
+// Third party dependencies
+from openzeppelin.token.erc20.library import ERC20
```

Comment:
@ClementWalter 
Because the contract_account initialization now performs a call to an ERC20 and to kakarot,
a lot of the other unit tests fail.
One solution is changing the unit tests like in this example `tests/unit/src/kakarot/instructions/test_system_operations.cairo`
But it is not very pretty. Do you have an alternative idea/solution that you would prefer?